astro/src/content/quickstarts/quickstart-javascript-nextjs-web.mdx

```diff
@@ -0,0 +1,277 @@
+
+On first load of Next.js this file will make sure that you have all of the correct environment variables. The variables are then exported in an object called `authOptions` which can be imported on the server when you need to get our session using `getServerSession`.
+
+The `FusionAuthProvider` is then provided to `NextAuth` as a provider for any `GET` or `POST` commands that are sent to the `/api/auth/*` route.
+
+`src/app/api/auth/[...nextauth]/route.ts`
+<RemoteCode url={frontmatter.coderoot + "/complete-application/src/app/api/auth/[...nextauth]/route.ts"}
+  lang="ts" />
+
+#### Styles
+
+Create a new file `src/app/globals.css` copy the below CSS for the changebank application.
+
+`src/app/globals.css`
+<RemoteCode url={frontmatter.coderoot + "/complete-application/src/app/globals.css"}
+  lang="ts" />
+
+#### Login Button
+
+Create a new file in `src/components/login-button.tsx` that will be used for a button component. Our login button will only be used on the client side so make sure to add `use client` at the top of this file.
+For this button you can use the `signIn` and `signOut` functions from `next-auth/react`. By passing in the session from our pages you can determine if the `Log in` or `Log out` should be shown. (Yeah I know if we could have all decided on Log in or Sign in over the last 20 years that would have been great!)
+
+`src/components/login-button.tsx`
+<RemoteCode url={frontmatter.coderoot + "/complete-application/src/components/login-button.tsx"}
+  lang="ts" />
+
+#### Layout
+
+<Aside type="tip">
+If this is your first time using Next.js's App Router, I would highly suggest reading through at least the <a href="https://nextjs.org/docs/app/building-your-application/routing" target="_blank">Routing Fundamentals</a>
+</Aside>
+
+Below you will find the full code for the [Root Layout](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts#root-layout-required)
+
+This has the overall structure of our application, the other pages will be added where `{children}` is located.
+
+There are two items of note:
+1. Getting the session information - `const session = await getServerSession(authOptions);`
+1. Passing the session to the Login Button - `<LoginButton session={session} />`
+
+`src/app/layout.tsx`
+<RemoteCode url={frontmatter.coderoot + "/complete-application/src/app/layout.tsx"}
+  lang="ts" />
+
+#### Home Page
+
+Create a new file `src/app/page.tsx` which will have the Homepage details. Not much here just an image and another Login button.
+
+`src/app/page.tsx`
+<RemoteCode url={frontmatter.coderoot + "/complete-application/src/app/page.tsx"}
+  lang="ts" />
+
+
+#### Account Page (protected)
+
+Create a new file `src/app/account/page.tsx` which will have the Account details.
```

Comment:
Do I need to create a directory here? I think so.

Reply:
Is your thought that we should call out all directories and that the relative path is not enough? 🤷‍♂️
<img width="290" alt="image" src="https://github.com/FusionAuth/fusionauth-site/assets/139158618/abe757d6-6ce9-4bb6-a88e-a3ae9a115908">